.pre-commit-config.yaml

```diff
@@ -9,12 +9,14 @@ repos:
       - id: ruff
         args: [--fix, --show-fixes]
         types_or: [python, pyi, jupyter]
+      - id: ruff-format
+        types_or: [python, pyi, jupyter]
 
   - repo: https://github.com/adamchainz/blacken-docs
     rev: "1.16.0"
     hooks:
        - id: blacken-docs
-         additional_dependencies: [black==22.12.0]
+         additional_dependencies: [black==23.*]
```

Comment:
I recently got to know that `ruff` is now capable of formatting code blocks in documentation files (Markdown, reST, etc.) as well, should we consider removing `blacken-docs`? Feel free to convert this comment to an issue if yes

Reply:
I don't think ruff can do that yet, see - https://github.com/astral-sh/ruff/issues/8237 and https://github.com/astral-sh/ruff/issues/3792